relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -230,6 +231,10 @@ pub fn extract_transaction_metrics(
         sampling_metrics,
     )?;
 
+    if project_id.value() == 1 {
```

Comment:
Please make this a feature flag right away and set this on sentry side.

Reply:
I have to agree with @jan-auer  on this. Please, add the feature flag into this PR, it's a very minimal change, but will greatly improve the quality of life right away.